src/Common/src/Interop/Linux/Interop.Data.cs

```diff
@@ -1,12 +1,11 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Data.Common
+internal static partial class Interop
 {
-    internal static class ExternDll
+    internal static partial class Data
```

Comment:
This Data class should not be needed.  The Odbc32 const should just be added to Interop.Libraries.